packages/manager/modules/iam/src/components/resourceSelect/translations/Messages_fr_FR.json

```diff
@@ -2,5 +2,5 @@
   "iam_resource_select_resource_types_error_required": "Veuillez sélectionner un ou plusieurs types de produits.",
   "iam_resource_select_resource_types_error_resources": "Aucune ressource n'est disponible pour ce ou ces types de produits.",
   "iam_resource_select_resources_error_required": "Veuillez sélectionner une ou plusieurs ressources",
-  "iam_resource_select_resources_error_limit": "Vous ne pouvez pas sélectionner plus de 10 ressources par politique."
+  "iam_resource_select_resources_error_limit": "Vous ne pouvez pas sélectionner plus de {{max}} ressources par politique."
```

Comment:
Not possible have 1 ressources max ? Else we show : "1 ressources"

Reply:
Nope, it's at least 10 for policy creation (and now 100 for group creation)